src/main/java/bio/terra/cli/businessobject/Server.java

```diff
@@ -8,13 +8,15 @@
 import bio.terra.cli.utils.FileUtils;
 import bio.terra.cli.utils.JacksonMapper;
 import bio.terra.datarepo.model.RepositoryStatusModel;
+import bio.terra.workspace.model.CloudPlatform;
```

Comment:
I'm still seeing [these errors](https://github.com/DataBiosphere/terra-cli/pull/366#pullrequestreview-1209326445)

Reply:
This should ideally never show up as long as the server config file is set up properly, and errors with the file would show up during deployment (which might not be convenient). 

The assumption here was that GCP, even if default has to be specified on the server config as supported. This way we decouple the config from the implementation. 
For the sake of UX we pick a default value, but that might not be expected on the infra. 

To avoid exceptions as these, initializing it to empty set. 

Could you share the server name used for this? and confirm that the corresponding server has the platforms enabled?

Reply:
Hmm I was doing `./gradlew installDist`. If I do `source tools/local-dev.sh` (as you suggested), it works, thanks.